AutomatedTesting/Gem/PythonTests/assetpipeline/fbx_tests/fbx_tests.py

```diff
@@ -387,7 +387,7 @@ def test_FBXBlackboxTest_AssetInfoModified_AssetReprocessed_ResultInExpectedProd
         self.run_fbx_test(workspace, ap_setup_fixture,
                           asset_processor, project, blackbox_param, True)
 
-
+    @pytest.mark.SUITE_sandbox
```

Comment:
you shouldn't need these on helper functions

Reply:
Updated and removed the unnecessary marks